### ISSUE TITLE
Red test reported as green if something was printed to console

### ISSUE THREAD

**Issue by Adambl4** [raniejade/spek-idea-plugin#48]
```
object Foobar : Spek({
    it("should be red") {
        print("qwerty")
        throw Exception()
    }
})
```
![image](https://user-images.githubusercontent.com/18431607/29374251-6279b8e6-82b9-11e7-8181-bbca82ea469f.png)

Android Studio 3.0 Canary 6
Spek plugin 0.4.1

**Comment by raniejade** [comment]
Working on a fix now thanks!

**Comment by raniejade** [comment]
Fixed in `v0.4.2`
